tests/accesscontrol/tests/access_control_crd_roles.go

```diff
@@ -0,0 +1,151 @@
+		err = globalhelper.CreateRole(testRole)
+		Expect(err).ToNot(HaveOccurred())
+
+		By("Start lifecycle-crd-scaling test")
+		err = globalhelper.LaunchTests(tsparams.TnfCrdRoles,
+			globalhelper.ConvertSpecNameToFileName(CurrentSpecReport().FullText()))
+		Expect(err).ToNot(HaveOccurred())
+
+		By("Verify test case status in Junit and Claim reports")
+		err = globalhelper.ValidateIfReportsAreValid(tsparams.TnfCrdRoles, globalparameters.TestCasePassed)
+		Expect(err).ToNot(HaveOccurred())
+	})
+
+	It("Custom resource is deployed, one role defined with multiple api groups [negative test]", func() {
+		By("Create a scale custom resource")
+		_, err := crdutils.CreateCustomResourceScale(tsparams.TnfCustomResourceName, randomNamespace,
+			tsparams.TnfTargetOperatorLabels, tsparams.TnfTargetOperatorLabelsMap)
+		Expect(err).ToNot(HaveOccurred())
+
+		By("Create a role for the custom resource")
+		testRole := globalhelper.DefineRole("memcached-role", randomNamespace)
+		globalhelper.RedefineRoleWithAPIGroups(testRole, []string{tsparams.TnfCustomResourceAPIGroupName, "rbac.authorization.k8s.io"})
+		globalhelper.RedefineRoleWithResources(testRole, []string{tsparams.TnfCustomResourceResourceName})
+		err = globalhelper.CreateRole(testRole)
+		Expect(err).ToNot(HaveOccurred())
+
+		By("Start lifecycle-crd-scaling test")
+		err = globalhelper.LaunchTests(tsparams.TnfCrdRoles,
+			globalhelper.ConvertSpecNameToFileName(CurrentSpecReport().FullText()))
+		Expect(err).To(HaveOccurred())
+
+		By("Verify test case status in Junit and Claim reports")
+		err = globalhelper.ValidateIfReportsAreValid(tsparams.TnfCrdRoles, globalparameters.TestCaseFailed)
+		Expect(err).ToNot(HaveOccurred())
+	})
+
+	It("Custom resource is deployed, one role with multiple resources defined [negative test]", func() {
+		By("Create a scale custom resource")
+		_, err := crdutils.CreateCustomResourceScale(tsparams.TnfCustomResourceName, randomNamespace,
+			tsparams.TnfTargetOperatorLabels, tsparams.TnfTargetOperatorLabelsMap)
+		Expect(err).ToNot(HaveOccurred())
+
+		By("Create a role for the custom resource")
+		testRole := globalhelper.DefineRole("memcached-role", randomNamespace)
+		globalhelper.RedefineRoleWithAPIGroups(testRole, []string{tsparams.TnfCustomResourceAPIGroupName})
+		globalhelper.RedefineRoleWithResources(testRole, []string{tsparams.TnfCustomResourceResourceName, "pods"})
+		err = globalhelper.CreateRole(testRole)
+		Expect(err).ToNot(HaveOccurred())
+
+		By("Start lifecycle-crd-scaling test")
+		err = globalhelper.LaunchTests(tsparams.TnfCrdRoles,
+			globalhelper.ConvertSpecNameToFileName(CurrentSpecReport().FullText()))
+		Expect(err).To(HaveOccurred())
+
+		By("Verify test case status in Junit and Claim reports")
+		err = globalhelper.ValidateIfReportsAreValid(tsparams.TnfCrdRoles, globalparameters.TestCaseFailed)
+		Expect(err).ToNot(HaveOccurred())
+	})
+
+	It("Custom resource is deployed, with improper role [negative test]", func() {
```

Comment:
For skip cases the convention is to end the testcase name with [skip], as in:
```suggestion
	It("Custom resource is deployed, with improper role [skip]", func() {
```